legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-bigquery/legend-engine-xt-relationalStore-bigquery-wrapper/src/main/java/com/deloitte/legend/engine/bigquery/BigQueryDriver.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Google LLC
+ */
+
+package com.deloitte.legend.engine.bigquery;
+
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.DriverPropertyInfo;
+import java.sql.SQLException;
+import java.sql.SQLFeatureNotSupportedException;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+public class BigQueryDriver implements Driver 
```

Comment:
Can we a simple Junit test that makes a connection and asserts that the connection is valid ?

We can provide the credentials and BigQuery endpoint coordinates to connect.